### DISCUSSION TITLE
upgrade question

### DISCUSSION THREAD

**Discussion by ap-coder** [spatie/laravel-medialibrary#2228] [Q&A]
for moving from 7 to 8 I see 2 columns (uuid, conversion_disk) that need to be filled in not just one and if you could give an example of where to use this that would be great. 
```
use Spatie\MediaLibrary\MediaCollections\Models\Media;
Media::cursor()->each(
   fn (Media $media) => $media->update(['uuid' => Str::uuid(), 'conversion_disk' => 'public'])
);
```
I see the snippet but I also added the 2nd field that was not filled in. 

Still you will have the change from **App\{{MODEL}}** to **App\Models\{{MODEL}}** in modelType to fix also. Is there a way to update that with a function also?

I have been attempting a laravel/console/command to fix these issues and here is what I have so far.  https://gist.github.com/ap-coder/4e02041152272ab584d26251e1d01f45

Any guidance would be greatly appreciated.

**Comment by 4unkur** [comment]
You can run this:
```php
        DB::table('media')->update([
            'conversions_disk' => DB::raw('disk')
        ]);
```
